sdk/go.mod

```diff
@@ -35,6 +35,7 @@ require (
 	github.com/hashicorp/go-version v1.6.0
 	github.com/hashicorp/golang-lru v0.5.4
 	github.com/hashicorp/hcl v1.0.1-vault-5
+	github.com/hashicorp/vault v1.13.2
```

Comment:
Are we sure we want to import the `vault` package into the sdk for a test?

Reply:
I think we should get an approval from someone in Core before we do this just to be sure. cc @ncabatoff

Reply:
Good call. Didn't look at this close enough. I can add in string generator helper functions in this package if we do decide that we don't want to pull them in from the Vault helper package

Reply:
Agreed, this doesn't look right. I'm surprised Go didn't complain about a circular dependency issue.

Reply:
I added some small functions in the template test file itself, so the circular dependency should no longer be an issue